### ISSUE TITLE
feature: parallel and staggered

### ISSUE THREAD

**Issue by jjhesk** [ericaltendorf/plotman#286]
is that possible to make more options for parallel and staggered schedule?

**Comment by BasilHorowt** [comment]
@jjhesk there is a lot of flexibility right now, what are you looking for?

**Comment by jjhesk** [comment]
not just one by one plotting

**Comment by BasilHorowt** [comment]
@jjhesk yes that is the point of plot man! No one is plotting 1 by 1 with it. Going to close this but I suggest reading the readme, and some items in the wiki (like the FAQ )

**Comment by qeeqez** [comment]
@jjhesk 
Use N temp dirs. Set **tmpdir_max_jobs=1**
In this case you will have Nx1 jobs running in parallel.
Also, you could use 1 temp dir and set tmpdir_max_jobs=N.

"there is a lot of flexibility right now"

**Comment by altendky** [comment]
@jjhesk various people have asked about simultaneously starting multiple plots.  I have not managed to get an explanation from any as to why 3x every 15 minutes would be better than 1x every 5 minutes.  What makes you interested in being able to do this?
